contracts/AirdropClaim.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+
+import "./libraries/Ownable.sol";
+import "./interfaces/IBEP20.sol";
+
+
+contract AirdropClaim is Ownable {
+
+    uint public claimExpiredAt;
+    mapping (address => uint) public claimWhitelist;
+
+    address public airdropTokenAddress;
+    
+    uint public registeredId = 0;
+    event Registered(uint registeredId_, address add_, uint amount_);
+
+    uint public claimId = 0;
+    event Claimed(uint claimId_, address add_, uint amount_);
+    
+
+    constructor(address airdropToken_) {
+        require(airdropToken_ != address(0));
+        airdropTokenAddress = airdropToken_;
+    }
+
+    receive () external payable {}
+    fallback () external payable {}
+
+    function pullFunds(address tokenAddress_) onlyOwner public {
```

Comment:
How about adding a few words as comments to explain the purpose of the function?

Reply:
done